lib/Dialect/FIRRTL/FIRRTLOpInterfaces.cpp

```diff
@@ -98,6 +98,31 @@ LogicalResult circt::firrtl::verifyModuleLikeOpInterface(FModuleLike module) {
   if (module->getNumRegions() != 1)
     return module.emitOpError("requires one region");
 
+  if (module.isPublic()) {
+    for (auto &pi : module.getPorts()) {
```

Comment:
nit: this function has many other checks, how about moving this one to a function whose name also captures its functionality?

Reply:
Oh, great call, thanks! Will refactor accordingly :+1: